All_Relationships/utils/disease_gene_lf.py

```diff
@@ -132,25 +133,35 @@ def LF_CHECK_DISEASE_TAG(c):
     """
     sen = c[0].get_parent()
     disease_name = re.sub("\) ?", "", c[0].get_span())
+    disease_name = re.sub(r"(\w)-(\w)", r"\g<1> \g<2>", disease_name)
```

Comment:
If speed is an issue, i.e. the regexes are the slowest part, you may consider compiling the regexes.